update(blog)!: CryptPad Review (#2880)
privacyguides/privacyguides.org: blog/posts/cryptpad-review.md

There are many reasons that CryptPad is a great tool and a perfect replacement to Google Drive. Let's dig into its pros and cons, why you should use it, why you can trust it, and how you can keep it secure.

## What makes CryptPad trustworthy

Why trust CryptPad more than Google? First, because its code is open, what CryptPad promises can be verified. Making source code open doesn't magically make it secure, but CryptPad is a project that had many eyes on in GitHub, has been around for over 10 years, and is actively maintained by a team experienced with open-source software. These are all elements that help building trust.

Second, all content in CryptPad is end-to-end encrypted [using strong algorithms](#privacy-and-encryption). When well implemented, this means the content of your files cannot be accessed by CryptPad itself. This greatly lowers the level of trust required: you do not have to trust the organization with your content, since it can't open it in the first place. If there was a data breach, and your encrypted content was stolen, your content would still be protected as long as you've used a unique, complex, and long password and protected it well.

> Additionally, CryptPad's flagship instance is hosted in France. This isn't magical either, but because European Union countries are all subjected to the GDPR most have adopted stronger data privacy practices by default. This also helps building trust.

Author: I agree, this is a great modification and makes this section flow much better. I also agree it's a good idea to emphasize the importance of E2EE and give more context about Google. Thank you! 

Small addition: there's a missing opening bracket "[" for the link on line 59.

<small aria-hidden="true">Unless credited otherwise, all screenshots from: Privacy Guides</small>

<div class="admonition danger" markdown>
<p class="admonition-title">Password security</p>

Make sure you note your unique, long, and complex password and username in a safe place. Due to the nature of end-to-end encryption, if you lose your username or password, there will be **no account recovery possible** and all your files will be lost.

</div>

### Basic security

Once you've created an account, the first thing you should do is secure it with a [second factor of authentication](https://www.privacyguides.org/en/basics/multi-factor-authentication/). To do this, click on your username on the upper-right, then select "Settings" > "Security & Privacy" and follow the instructions for "Two-Factor Authentication".

On the same "Settings" page, depending on your [threat model](https://www.privacyguides.org/en/basics/threat-modeling/), you might also want to enable "Safe Links", disable "Feedback" (disabled by default), and disable "Cache". The cache option can be very convenient if the device you use isn't at risk, however.

### Destroying data

> Again on this same page, CryptPad offers a quick option to "Destroy all owned documents" at once. This can be a lifesaver feature for people with higher threat models. Take good note of where this feature is in the menu.

Reviewer: ```suggestion
Another option that CryptPad offers on the "Settings" page is a quick way to "Destroy all owned documents" at once. This can be a lifesaver feature for people at particularly high risk. Take good note of where this feature is in the menu.
```

With solid end-to-end encryption for all your document content and minimal data collection at all steps of the registration process, CryptPad is an excellent option for your data privacy *and* your data security.

If data privacy is important to you, **there is no reason to keep your sensitive information in the Google ecosystem**, especially with everything that we know about it now.

Switching to CryptPad, you will benefit from the same type of applications (and more), still be able to share links with external parties without requiring them to create an account, and store information remotely in a secure way.

But *in addition* to Google Drive, you will benefit from strong end-to-end encryption on everything you wish to keep private, true document destruction for everything that you want gone, and a complete office suite that only you and your team can access, away from intrusive Google scans or even CryptPad's knowledge.

If you need a remote collaborative office suite that is secure and respects your privacy, there is no reason not to switch to CryptPad.

## Support CryptPad

> If you wish to support this free and open-source project, you can help CryptPad with:
> 
> Donations: [https://opencollective.com/cryptpad/contribute/](https://opencollective.com/cryptpad/contribute/)
> 
> Contributions: [https://github.com/cryptpad](https://github.com/cryptpad)
> 
> Follow CryptPad Mastodon: [https://fosstodon.org/@cryptpad](https://fosstodon.org/@cryptpad)
> 
> Watch CryptPad's channel on PeerTube: [https://peertube.xwiki.com/c/cryptpad_channel/videos](https://peertube.xwiki.com/c/cryptpad_channel/videos)

Reviewer: ```suggestion
If you wish to support this free and open-source project, you can help CryptPad by:

- [Donating on Open Collective](https://opencollective.com/cryptpad/contribute/)

-[Contributing on GitHub](https://github.com/cryptpad)

- [Following CryptPad on Mastodon](https://fosstodon.org/@cryptpad)

- [Watching CryptPad's videos on PeerTube](https://peertube.xwiki.com/c/cryptpad_channel/videos)
```
